### PR TITLE
Device administrator vs Device owner #29

### PR DESCRIPTION
**Pull request by dmwheel1**
Address definition of Device User, Device Owner and Device Administrator
This should address and close Issue #29

### CODE REVIEW

File: draft-ietf-teep-architecture.md [hunk 1]
```diff
@@ -195,10 +195,24 @@ The following terms are used:
     A Service Provider requires the help of a TAM in order to provision
     the Trusted Applications to remote devices.
 
-  - Device Administrator:  An entity that owns or is responsible for
-    administration of a Device. A Device Administrator has privileges
-    on the Device to install and remove applications and TAs, approve
-    or reject Trust Anchors, and approve or reject Service Providers,
+  - Device User: A human being that uses a device. Many devices have
+    a single device user. Some devices have a primary device user with
+    other human beings as secondary device users (e.g. parent allowing
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]: nit: add comma after "e.g."

File: draft-ietf-teep-architecture.md [hunk 2]
```diff
@@ -195,10 +195,24 @@ The following terms are used:
...
+    children to use their tablet or laptop). Relates to Device Owner
+    and Device Administrator.
+
+  - Device Owner: A device is always owned by by someone. It is common for
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]: typo: "by by"

**Review by dthaler — APPROVED** [review]
Ok other than typos

File: draft-ietf-teep-architecture.md (hunk 2 above)
Comment (dmwheel1) [reply draft-ietf-teep-architecture.md]: Fixed

File: draft-ietf-teep-architecture.md (hunk 1 above)
Comment (dmwheel1) [reply draft-ietf-teep-architecture.md]: fixed

**Pull request merged**